diff -aurBb /home/httpd/htdocs/users/vashkevich/magento-prepare/var/ait_patch_orig/compare12812415976674a0146e3b3c93 /home/httpd/htdocs/users/vashkevich/magento-prepare/var/ait_patch_modif/compare12812415976674a0146e3b3c93
--- /home/httpd/htdocs/users/vashkevich/magento-prepare/var/ait_patch_orig/compare12812415976674a0146e3b3c93	2009-05-06 11:14:27 +0300
+++ /home/httpd/htdocs/users/vashkevich/magento-prepare/var/ait_patch_modif/compare12812415976674a0146e3b3c93	2009-05-06 11:14:27 +0300
@@ -48,7 +48,10 @@
                 <th><?php echo $this->helper('sales')->__('Subtotal') ?></th>
                 <th><span class="nobr"><?php echo $this->helper('sales')->__('Tax Amount') ?></span></th>
                 <th><span class="nobr"><?php echo $this->helper('sales')->__('Tax Percent') ?></span></th>
+                <?php /*
                 <th><span class="nobr"><?php echo $this->helper('sales')->__('Discount Amount') ?></span></th>
+                */ ?>
+                <th><span class="nobr"><?php echo $this->helper('sales')->__('Discount(-)/Surcharge(+) Amount') ?></span></th>
                 <th class="last"><span class="nobr"><?php echo $this->helper('sales')->__('Row Total') ?></span></th>
             </tr>
         </thead>
